packages/dds/merge-tree/src/revertibles.ts

```diff
@@ -281,11 +281,28 @@ function revertLocalInsert(
 	}
 }
 
+/**
+ * Revertibles are new and require the option
```

Comment:
More of a general comment for this file, but can we either update these comment blocks to add more information specific to the functions or remove them?